client/src/index.js

```diff
@@ -2,17 +2,20 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import { Provider } from "react-redux";
 import { BrowserRouter as Router } from "react-router-dom";
+import { CookiesProvider } from "react-cookie";
 import App from "./App";
 import store from "./Store";
 import Theme from "./Theme";
 
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
   <Theme>
-    <Router>
-      <Provider store={store}>
-        <App />
-      </Provider>
-    </Router>
+    <CookiesProvider>
```

Comment:
why are you using this module,
Can you get cookies from js as `document.cookies`?
if not check this module I think it's better than the one you used
[js-cookie](https://www.npmjs.com/package/js-cookie)

Reply:
I want to use [js-cookie](https://www.npmjs.com/package/js-cookie)